datahub-actions/src/datahub_actions/cli/actions.py

```diff
@@ -99,14 +117,22 @@ def run(ctx: Any, config: List[str], debug: bool) -> None:
                 pipeline_config_to_pipeline(pipeline_config_dict)
             )  # Now, instantiate the pipeline.
 
+    # Exit early if no valid pipelines were created
+    if not pipelines:
+        logger.error(
+            f"No valid pipelines were started from {attempted_configs} config(s). "
```

Comment:
we really don't need `attempted_configs`
```suggestion
            f"No valid pipelines were started from {len(configs)} config(s). "
```

Reply:
good call - yes!